Draw red rectangles around the faces

skalpin opened the pull request:
Drawing rectangles around the faces was added to mimic the Cognitive API tutorial for C#.

azurecla commented:
Hi __@ahmetalpbalkan__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ahmetb). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
Changes reviewed and accepted by SME.

skalpin committed:
Draw red rectangles around the faces

Drawing rectangles around the faces was added to mimic the Cognitive API tutorial for C#.

GitHubber17 committed:
Update FaceAPIinPythonTutorial.md
